### DISCUSSION TITLE
Version dependency issue : Monaco - MonacoLanguageClient

### DISCUSSION THREAD

**Discussion by GSri30** [TypeFox/monaco-languageclient#505] [General]
I was going through this version dependency table
`https://github.com/TypeFox/monaco-languageclient#monaco-editor--monaco-vscode-api-compatibility-table`.

I'm currently using `"monaco-editor": "^0.25.2"`. It's not listed in the above table. What would be the monaco-languageclient version suitable for this?

**Comment by CGNonofr** [comment]
Out of curiosity, is there a reason to use such a old version? it was released more than 2 years ago and this library has received a lot of improvements since then

**Reply by GSri30** [reply]
Yeah. We are trying to integrate this into our org. And we have been using this particular version of the editor.
